src/test/java/homework_2/RandomCharsTableTest.java

```diff
@@ -0,0 +1,37 @@
+package homework_2;
+
+import base.UnitBase;
+import homework_2.random_chars_table.RandomCharsTable;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class RandomCharsTableTest extends UnitBase {
```

Comment:
Most of your tests are failing. Please check

Reply:
![image](https://user-images.githubusercontent.com/58872509/134569261-3cebfc09-249d-4b18-a3bb-3849046e6248.png)